guide/14-deep-learning/How CycleGAN Works.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<p>Figure 1. Overview of CycleGAN architecture <code>translating from image to map domain</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>